modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -57,4 +57,16 @@ $ oc get clusteroperator image-registry
 //type: Degraded
 //----
 
-​​
+[Note]
```

Comment:
If you do keep this in a note, this has to be "NOTE" all caps, otherwise it doesn't render quite right when built. (Same with the below module).